From cb5ebbf08a4dbd9e610ea3beded3b215233fda55 Mon Sep 17 00:00:00 2001
From: Vishesh Handa <me@vhanda.in>
Date: Tue, 29 Apr 2014 15:35:59 +0200
Subject: [PATCH 28/28] FileIndexingJob: Do not try to start the process again

This happened in the case when the job would be paused and then resumed
after that while the process was still running.
---
 src/file/fileindexingjob.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/file/fileindexingjob.cpp b/src/file/fileindexingjob.cpp
index 758d280..a62636d 100644
--- a/src/file/fileindexingjob.cpp
+++ b/src/file/fileindexingjob.cpp
@@ -170,7 +170,8 @@ bool FileIndexingJob::doResume()
         return false;
 
     m_suspended = false;
-    start(m_args);
+    if (!m_process)
+        start(m_args);
     return true;
 }
 
-- 
1.9.0

